.github/workflows/get-merge-commit.yml

```diff
@@ -5,6 +5,14 @@ on:
     paths:
       - .github/workflows/get-merge-commit.yml
```

Comment:
Might want to consider removing this now that this requires inputs (and apparently defaults don’t apply to non-`workflow_run` triggers…).